### PR TITLE
Feat: Products

### PR DESCRIPTION
**Pull request by dikshadhote**
Following Feature are added :
1. Sidebar having filter UI
2.  Product list rendering dynamically with api data
3. Scrollable product list

image

Preview : https://deploy-preview-4--dreamstoreproducts.netlify.app/products

### CODE REVIEW

File: src/components/products/FilterProduct.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <label htmlFor="checkbox-bed">Beds & mattresses</label>
+          </li>
+          <li>
+            <input
+              id="checkbox-decor"
+              className="form-check-input"
+              type="checkbox"
+            />
+            <label htmlFor="checkbox-decor">Storage & Organisation</label>
+          </li>
```
Comment (JantuDeb) [inline src/components/products/FilterProduct.js]: You can use an array to store all the categories then `map()` over it

File: src/components/products/FilterProduct.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          <li>
+            <input
+              id="radio-one"
+              className="form-check-input"
+              type="radio"
+              name="rating"
+            />
+            <label htmlFor="radio-one">1 stars & above</label>
+          </li>
```
Comment (JantuDeb) [inline src/components/products/FilterProduct.js]:
```
[4,3,2,1].map(star=><li>
            <input
              id="radio-one"
              className="form-check-input"
              type="radio"
              name="rating"
            />
            <label htmlFor="radio-one">{star} stars & above</label>
          </li>)
```

**Review by JantuDeb — APPROVED** [review]
The code looks good,  added some comments.

File: src/components/products/FilterProduct.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+export default function FilterProduct() {
+  const categories = [
+    "Sofas & armchairs",
+    "Cookware & Table ware",
+    "Beds & mattresses",
+    "Storage & Organisation",
+  ];
+
+  const rating = [4, 3, 2, 1];
```
Comment (t007rushi) [inline src/components/products/FilterProduct.js:11]: you can put in separate .js file

File: src/components/products/Products.js [hunk 4]
```diff
@@ -1,5 +1,77 @@
...
+import axios from "axios";
+import { products } from "../../backend/db/products";
+import FilterProduct from "./FilterProduct";
 
 export default function Products() {
-  return <div>Products</div>;
+  async function apiCall() {
+    const { data } = await axios.get("/api/products");
+    const { products } = data;
+    console.log(products);
```
Comment (t007rushi) [inline src/components/products/Products.js]: remove console before push

**Review by t007rushi — APPROVED** [review]
some minor changes for ahead

**Pull request merged**